### ISSUE TITLE
Training stuck at the end of the first epoch

### ISSUE THREAD

**Issue by XxUpUp** [milesial/Pytorch-UNet#505]
Hi, I'm having the problem that when the last batch of my first epoch is trained, the program can't start the next epoch but stays stuck there and no error message appears, what could be the reason for this? (Note: I have made some changes to train.py, but not the main body of the code)
![image](https://github.com/user-attachments/assets/175c04c6-7111-4edb-9fd7-7abb554264fe)

**Comment by zaoyueri** [comment]
什么原因呢？

**Comment by XxUpUp** [comment]
> 什么原因呢？

I'm still not quite sure what the cause of this problem is.

**Comment by chenshans** [comment]
请问你解决了吗，我也遇到了一模一样的问题

**Comment by XxUpUp** [comment]
> 请问你解决了吗，我也遇到了一模一样的问题

Not yet.

**Comment by ZafirTan** [comment]
You can try use PyTorch 1.13 or later, I solve it by change pytorch verson

**Comment by cvliu-hub** [comment]
请问解决了吗 我也遇到了一样的问题

**Comment by XxUpUp** [comment]
> 请问解决了吗 我也遇到了一样的

**Comment by XxUpUp** [comment]
> 请问解决了吗 我也遇到了一样的问题

Perhaps you could try adjusting the learning rate, for example modifying it from 0.0001 to 0.001
